### ISSUE TITLE
IDE with flutter plugin alleges dependency to be non-existent

### ISSUE THREAD

**Issue by hu1buerger** [flutter/flutter-intellij#5819]
```dart 
import 'package:flutter_gen/gen_l10n/app_localizations.dart'; // Android studio alleges this to be non existent
import 'package:flutter_localizations/flutter_localizations.dart';

class App extends StatelessWidget{
  
   [....]

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      restorationScopeId: 'app',

      localizationsDelegates: const [
        AppLocalizations.delegate, // the usage of the import
        GlobalMaterialLocalizations.delegate,
        GlobalWidgetsLocalizations.delegate,
        GlobalCupertinoLocalizations.delegate,
      ],
      supportedLocales: const [
        Locale('en', ''), // English, no country code
      ],

      onGenerateTitle: (BuildContext context) => AppLocalizations.of(context)!.appTitle,
      theme: ThemeData.dark(),

     [...]

    );
  }
}
```

TL;DR: Android studio / IntelliJ believes an import `import 'package:flutter_gen/gen_l10n/app_localizations.dart';` to be non-existent. The dart analysis window also shows the error.
But `$flutter analyse` disagrees and `flutter run` also disagrees and starts compiling and executing the code without issue.

For more info head over to https://github.com/flutter/flutter/issues/91756

**Comment by stevemessick** [comment]
According to the Flutter [localizations docs](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) you need to run the app to get code generation to execute. After running the app the analysis error disappears.

**Comment by Hu1buerger** [comment]
Shouldn't the error message reflect that?

**Comment by Sushma-Shrestha** [comment]
**Problem while self-learning:**

for this code, in official site of flutter:
https://gallery.flutter.dev/#/demo/app-bar

there is,
import 'package:flutter_gen/gen_l10n/gallery_localizations.dart';

for this i have tried installing package flutter_gen, flutter_intl, localization.
upgrade, clean, run.
Also, saved the l10n.yaml file in lib.

However, it stilll shows error, may be because i am starter and messed up,
Can someone tell me which package has to be installed exactly and process? **PLEASE!**

I am self-learning and don't know what to do!!

**Thankyou for viewing my comment!**

**Comment by theRealCocapepsi** [comment]
I just tried `flutter gen-l10n ` then it worked
